src/AzureExtension/DataModel/DataObjects/Identity.cs

```diff
@@ -42,9 +44,9 @@ public class Identity
 
     public string Avatar { get; set; } = string.Empty;
 
-    // Represents whether this identity is associated with a DeveloperId that is logged in.
-    // This is the backing database column for the IsLoggedInDeveloper property.
-    public long IsDeveloper { get; set; } = DataStore.NoForeignKey;
+    // The DeveloperLoginId associated with this identity, if one exists.
+    // Empty means there is no associated LoggedInDeveloperId.
```

Comment:
Nit just because "null" also feels kind of "empty" to me and I like when things are super obvious.
```suggestion
    // Empty string means there is no associated LoggedInDeveloperId.
```

Reply:
This is because the sql table is defined as not null and I don't like nullable types unless necessary.  I'll see about changing it to nullable type and whether that is significant churn.